UnitTests/fixtures/colors.txt

```diff
@@ -1,3 +1,5 @@
+# Hey this is a comment
+# And another one
```

Comment:
Maybe also insert dummy comment lines in between valid ones to make the unit test even more generic?